OpenDreamRuntime/LocalHostConGroup.cs

```diff
@@ -42,6 +50,13 @@ public sealed class LocalHostConGroup : IConGroupControllerImplementation, IPost
             return Equals(addr, IPAddress.Loopback) || Equals(addr, IPAddress.IPv6Loopback);
         }
 
+        public bool CheckInvokable(CommandSpec command, ICommonSession? user, out IConError? error)
+        {
+            error = null;
+            if (user is null) return false;
```

Comment:
Looking at [SS14's implementation](https://github.com/space-wizards/space-station-14/blob/4fb46c1cb69efa6a46a8edeab47689b9b4c7ba77/Content.Server/Administration/Managers/AdminManager.cs#L454) shows that `user` being null implies it's the server console. The server console should always be able to execute a command.
```suggestion
            if (user is null) return true; // Server console
```